testproxy/services/read-rows.js

```diff
@@ -72,7 +72,13 @@ const readRows = ({clientMap}) =>
         rows: rows.map(getRowResponse),
       };
     } catch (e) {
-      return {status: e};
+      return {
+        status: {
+          code: e.code,
+          details: [],
```

Comment:
```suggestion
          details: [], // e.details must be in an empty array for the test runner to return the status, work to fix the test runner to properly return details is tracked in b/xxxxxxx
```

Reply:
Committed and https://b.corp.google.com/issues/383096533 added.